pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.10"
 poetry = "^1.7.1"
-reqstool-python-decorators = "^0.1.dev6" #Change later when proper release is made. 
```

Comment:
We should set an explicit version (did we do that with hatch?) and not a range.

See: https://python-poetry.org/docs/dependency-specification/#caret-requirements

Reply:
Yes, we are targeting specific versions in the Hatch plugin. This is now fixed here as well

Reply:
Think we can close this PR as the version has been updated in PR #12